src/constants/api/misc.ts

```diff
@@ -0,0 +1,7 @@
+let origin = '/api';
```

Comment:
Codacy has a fix for the issue: [Replace `'/api'` with `"/api"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
let origin = "/api";
```